README.md

```diff
@@ -1,20 +1,109 @@
+[eRegulations](http://cfpb.github.io/eRegulations) is a web-based tool that makes regulations easier to find, read and understand with features such as inline official interpretations, highlighted defined terms, and a revision comparison view.
 
+eRegs is made up of three core components:
 
-### Vagrant Install
-1. `brew cask install vagrant`
-2. `vagrant up` to create and provision the VM
-3. `vagrant ssh` to connect to the running VM
-4. `cp /vagrant/regs_bootstrap.sh regs_bootstrap.sh && ./regs_bootstrap.sh` to clone directories and install dependencies
-5. Continue from step 3 above.
+* [regulations-parser](https://github.com/cfpb/regulations-parser): Parse regulations
+* [regulations-core](https://github.com/cfpb/regulations-core): Regulations API
+* [regulations-site](https://github.com/cfpb/regulations-site): Display the regulations
+
+This repository contains scripts that boostrap a coherent eRegulations working envionrment, either locally or in a [Vagrant](https://www.vagrantup.com/) virtual machine.
+
+## Bootstrap Locally
+
+To bootstrap a local (i.e. not virtualized) eRegs environment, you'll
+need to install Python's virtualenv and virtualenvwrapper. 
+
+### Requirements
+
+Install [virtualenv](https://virtualenv.pypa.io/en/latest/):
+
+```shell
+pip install virtualenv
+```
+
+Then install and configure 
+[virtualenvwrapper](https://virtualenvwrapper.readthedocs.org/en/latest/). 
+More information on how virtualenvwrapper works and how you may wish to 
+set it up in your shell can be found in the 
+[virtualenvwrapper documentation](https://virtualenvwrapper.readthedocs.org/en/latest/install.html).
+
+```shell
+pip install virtualenvwrapper
+export WORKON_HOME=~/envs
+mkdir -p $WORKON_HOME
+source `which virtualenvwrapper.sh`
+```
+
+### Bootstrapping
+
+Once you have these requirements installed you can run the
+`regs_bootstrap.sh`:
+
+```shell
+./regs_bootstrap.sh
+```
+
+This will clone all of the eRegs repositories, make seperate virtualenvs 
+for them, and setup their dependencies.
+
+## Bootstrap with Vagrant
+
+To bootstrap eRegs in a [Vagrant](https://www.vagrantup.com/) virtual 
+machine, you'll need to install Vagrant locally. Everything else should 
+be taken care of by this repository's `Vagrantfile`.
+
+### Requirements
+
+Install [Vagrant](https://www.vagrantup.com/). On Mac OS X, this can be
+done with [Homebrew](http://brew.sh):
+
+```shell
+brew install vagrant
+```
+
+### Bootstrapping
+
+The eRegs bootstrapping process will be performed as part of the Vagrant
+provisioning process. All one needs to do is provision the Vagrant
+virtual machine. 
+
+```shell
+vagrant up
+```
+
+Once the virtual machine is running, you should be able to access
+regulations-core at [http://localhost:8000](http://localhost:8000) and 
+regulations-site at [http://localhost:8001](http://localhost:8001).
+
+You can also connect to the virtual machine via SSH.
+
+```shell
+vagrant ssh
+```
+
+## Using
```

Comment:
It shouldn't hold up merging this, but should we give instructions for running the parser locally as well?